changelog/_unreleased/2020-09-17-next-10854-support-custom-cache-folders.md

```diff
@@ -0,0 +1,16 @@
+---
+title: NEXT-10854
+issue: NEXT-10854
+author: Yann Karl
+author_email: y.karl@webweit.de 
+author_github: yannick80
+---
+# Core
+*  Renamed `cacheDir` to `projectDir`, updated write path for `dump` in `src/Core/Framework/Plugin/BundleConfigDumper.php`
```

Comment:
Please use `Added`, `Changed`, `Removed` or `Deprecated` as keywords here.

```suggestion
*  Changed property name `cacheDir` to `projectDir`, updated write path for `dump` in `src/Core/Framework/Plugin/BundleConfigDumper.php`
```

Reply:
also done, good to know for the next pr